README.md

```diff
@@ -1 +1,98 @@
-# Things3 Low-Level Python API to the Database
\ No newline at end of file
+Get your Things app data into a Python dict.
+
+## Install
+```python
+$ pip install things.py
```

Comment:
pip -> pip3?
things -> things3?

Reply:
- I think many of us have pip mapped to pip3. See flask, pandas, requests and other libraries.
- What happens once Things 3 becomes Things 4? It seems to me not having the version number in there makes it more future-proof.

Reply:
pip: I think on macOS (our main target OS) it's pip2 by default
things: you're right